2023/incoming/santa-kotlin-is-coming-to-perl.pod

```diff
@@ -0,0 +1,275 @@
+Author: jjatria@gmail.com
+Title: Santa Kotlin is coming to Perl
+Topic: FFI::Platypus
+
+=encoding utf8
+
+This post starts with a long introduction to explain I<why> I'm writing, but
+if you want to skip all of that you can jump directly to the
+L<meat and potatoes|/Binding to Kotlin from Perl>.
+
+=head2 Stay awhile and listen
```

Comment:
```suggestion
=head2 Stay a while and listen
```

Reply:
Ah, that was my first thought as well! But then I had second thoughts, because the only reason I know of this line is because it's one of the lines that [Deckard Cain](https://diablo.fandom.com/wiki/Deckard_Cain) says in the original Diablo game, and it looks like in that context it is [consistently styled](https://diablo.fandom.com/wiki/Stay_Awhile_and_Listen) like "stay _awhile_ and listen" (which I also find odd, but...).

Do you reckon I should still change it?

Reply:
> Do you reckon I should still change it?

No. I didn't realize it was a reference. That makes sense. Also, my inclination is not to edit articles too heavily. I don't want to redact all of the personality and charm.